webapp/app/[locale]/tunnel/transaction-history/_components/topBar.tsx

```diff
@@ -0,0 +1,57 @@
+import { featureFlags } from 'app/featureFlags'
+import { useTranslations } from 'next-intl'
+import { Tabs, Tab } from 'ui-common/components/tabs'
+
+import { ReloadHistory } from './reloadHistory'
+import { TunnelHistorySyncStatus } from './tunnelHistorySyncStatus'
+
+export type FilterOptions = 'all' | 'bitcoin' | 'ethereum'
+
+type Props = {
+  filterOption: FilterOptions
+  onFilterOptionChange: (filter: FilterOptions) => void
+}
+
+export const TopBar = function ({ filterOption, onFilterOptionChange }: Props) {
+  const t = useTranslations('tunnel-page.transaction-history.top-bar')
+  return (
+    <div className="flex w-full flex-wrap items-center gap-x-2 gap-y-2 px-3.5 py-2 md:flex-nowrap md:px-3">
+      <span className="order-1 flex-shrink-0 flex-grow basis-2/5 md:flex-grow-0 md:basis-auto">
```

Comment:
Perhaps it's better to use h1 or h2 here, thinking about web semantics. What are your thoughts on this?

Reply:
I updated it to use a `h5` [here]([27ca8ba](https://github.com/hemilabs/ui-monorepo/pull/522/commits/27ca8bae49cd783182d8c8cea905080c732d4000)). As this is a filter bar, this is in pair with the lowest headings we've used so far across the app (We haven't used `h6`)